exercises/concept/plane-tickets/plane_tickets_test.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+import pytest
+
+from plane_tickets import (
+    generate_seats,
+    assign_seats
+)
+
+class PlaneTicketsTest(unittest.TestCase):
+
+
+    @pytest.mark.task(taskno=1)
+    def test_task1_is_generator(self):  # * Tests if [Task 1] actually returns a generator.
+        input_vars = [5]
+        output = ["1A"]
+        for variant, (input_var, output) in enumerate(zip(input_vars, output), start=1):
+            with self.subTest(f"variation #{variant}", input_data=input_var, output_data=output):
+                self.assertEqual(generate_seats(input_var).__next__(), output)
```

Comment:
If this is a plain test to see that a generator was created, I think [`assertIsInstance`](https://docs.python.org/3/library/unittest.html?highlight=unittest#unittest.TestCase.assertIsInstance) might be better here.